src/LogTarget.php

```diff
@@ -1,8 +1,8 @@
 <?php
 /**
- * @link http://www.yiiframework.com/
+ * @link      http://www.yiiframework.com/
```

Comment:
These alignment changes are making review hard. Would you please revert these? Thanks.

Reply:
Complete!